installer/PowerToysSetup/Product.wxs

```diff
@@ -614,13 +638,35 @@
       </Component>
     </DirectoryRef>
 
+    <!-- Color Picker Resources -->
     <DirectoryRef Id="ColorPickerResourcesFolder" FileSource="$(var.BinX64Dir)modules\$(var.ColorPickerProjectName)\Resources">
       <Component Id="Module_ColorPicker_Resources" Guid="7544BD0F-1DB6-4C53-89D3-ADAD472FDCC1">
         <?foreach File in colorPicker.cur;icon.ico?>
         <File Id="ColorPickerFile_$(var.File)" Source="$(var.BinX64Dir)modules\$(var.ColorPickerProjectName)\Resources\$(var.File)" />
         <?endforeach?>
       </Component>
     </DirectoryRef>
+    
+    <!-- Espresso -->
+    <DirectoryRef Id="EspressoInstallFolder" FileSource="$(var.BinX64Dir)modules\$(var.EspressoProjectName)">
+      <Component Id="Module_Espresso" Guid="F26F5780-5B38-43B2-BC21-8406ED6E2071" Win64="yes">
+          <!-- 
+            EspressoModuleInterface.exp;
+            EspressoModuleInterface.lib;
+            EspressoModuleInterface.pdb;
+            ManagedCommon.pdb;
+            ManagedTelemetry.pdb;
+            Microsoft.PowerToys.Settings.UI.Lib.pdb;
+            PowerToys.Espresso.pdb;
+            PowerToys.Espresso.runtimeconfig.dev.json;
```

Comment:
This file is needed

Reply:
We include  PowerToys.Espresso.runtimeconfig.json;

there was no other instance in WSX of runtimeconfig.dev.json  Settings has this for instance.

Reply:
Let me verify to make sure.